electron-app/src/views/Dashboard.vue

```diff
@@ -1,46 +1,122 @@
 <template>
   <v-container>
     <v-row>
-      <v-col>
-        <h1 class="page-header">Dashboard</h1>
+      <v-col cols="12">
+        <overall-balances
+          :is-loading="blockchainIsLoading || exchangeIsLoading"
+        ></overall-balances>
       </v-col>
     </v-row>
-
-    <div class="dashboard__information-boxes">
-      <exchange-box
-        v-for="exchange in exchanges"
-        :key="exchange.name"
-        :name="exchange.name"
-        :amount="exchange.total"
-      ></exchange-box>
-      <information-box
-        v-if="!blockchainTotal.isZero()"
-        id="blockchain_box"
-        icon="fa-hdd-o"
-        :amount="blockchainTotal"
-        blockchain
-      ></information-box>
-      <information-box
-        v-if="!fiatTotal.isZero()"
-        id="banks_box"
-        icon="fa-university"
-        :amount="fiatTotal"
-      ></information-box>
-    </div>
-
-    <v-row>
-      <v-col cols="12">
-        <h1 class="page-header">All Balances</h1>
+    <v-row class="mx-0 flex" justify="center">
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="exchange"
+          can-refresh
+          :is-loading="exchangeIsLoading"
+          @refresh="refreshBalance($event)"
+        >
+          <div slot="tooltip">
+            Aggregate value of all balances<br />in each configured exchange.
+          </div>
+          <div v-if="exchanges.length < 1">
+            <v-card-actions>
+              <v-btn text color="primary" to="/settings/api-keys">
+                Add an exchange
+              </v-btn>
+            </v-card-actions>
+          </div>
+          <div v-else>
+            <exchange-box
+              v-for="exchange in exchanges"
+              :key="exchange.name + '_new'"
```

Comment:
is this new postfix really needed here?

Reply:
No it's not, it was left over from when it was exchange-box-new. Good catch!